recipes/symengine/all/conandata.yml

```diff
@@ -0,0 +1,7 @@
+sources:
+  "0.8.1":
+    url: "https://github.com/symengine/symengine/releases/download/v0.8.1/symengine-0.8.1.tar.gz"
+    sha256: "41eb6ae6901c09e53d7f61f0758f9201e81fc534bfeecd4b2bd4b4e6f6768693"
+  "0.9.0":
```

Comment:
Why are we using two different versions?

Reply:
We are only using 0.9.0 (as that is the version specified in the tket conanfile). We could delete the other option from here, but it does no harm (and is copied directly from the conan-center recipe). When new versions come out, we can just add them here and update tket retaining the ability to revert if there are problems.